quesma/quesma/schema_transformer.go

```diff
@@ -534,6 +560,7 @@ func (s *SchemaCheckPass) Transform(queries []*model.Query) ([]*model.Query, err
 	}{
 		{TransformationName: "PhysicalFromExpressionTransformation", Transformation: s.applyPhysicalFromExpression},
 		{TransformationName: "WildcardExpansion", Transformation: s.applyWildcardExpansion},
+		{TransformationName: "FieldEncodingTransformation", Transformation: s.applyFieldEncoding},
```

Comment:
I'd comment why the position of `FieldEncodingTransformation` should be here :)

Reply:
Will add comments